extractor/config/config.toml

```diff
@@ -21,6 +21,12 @@ main-class = "org.springframework.boot.loader.JarLauncher"
 read-manifest-of-executable-jar = true
 jar-version-manifest-entry = "Spring-Boot-Version"
 
+[[fingerprints.java]]
+runtime-name = "Spring Boot"
+main-class = "org.springframework.boot.loader.WarLauncher"
```

Comment:
this is not correct: it should be `org.springframework.boot.loader.launch.WarLauncher`

Reply:
yes, You are right. I will update it.